http.json

```diff
@@ -1094,9 +1094,264 @@
+                        "/web_storage/file.tgz"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.hostname",
+                    "data_type": "string",
+                    "contains": [
+                        "host name"
+                    ],
+                    "example_values": [
+                        "http://192.168.0.1"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary.exit_status",
+                    "data_type": "numeric",
+                    "example_values": [
+                        0
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.name",
+                    "data_type": "string",
+                    "example_values": [
+                        "file.tgz"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.size",
+                    "data_type": "numeric",
+                    "example_values": [
+                        412
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.vault_id",
+                    "data_type": "string",
+                    "contains": [
+                        "vault id"
+                    ],
+                    "column_order": 1,
+                    "column_name": "Vault ID",
+                    "example_values": [
+                        "dc871f816c4d179f3a3cea24b4bc81a67562c"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "column_order": 0,
+                    "column_name": "Result Message",
+                    "example_values": [
+                        "Transferred file"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "put file",
+            "description": "Put a file from the vault to another location",
+            "type": "generic",
+            "identifier": "put_file",
+            "verbose": "Provide the path to store the file on QA server. For example, <b>/web_storage/test_repo/</b>.",
```

Comment:
I don't think we should say the file specifically goes on a QA server. Something more generic would be better. Or, make the example use the QA server only

Reply:
I am making it as 'file server' which I guess is a generic term.